packages/components/psammead-section-label/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Psammead Section Label Changelog
+
+<!-- prettier-ignore -->
+| Version | Description |
+|---------|-------------|
+| 0.1.0-alpha.0 | [PR#449](https://github.com/BBC/psammead/pull/449) Create initial package with Section Label components. |
```

Comment:
```suggestion
| 0.1.0-alpha.1 | [PR#449](https://github.com/BBC/psammead/pull/449) Create initial package with Section Label components. |
```
Alpha versions usually start from `.1`

Reply:
do you have a reference for this? I don't see it mentioned on <https://semver.org>, and my understanding of the team's decision was to start at `.0`, and it was used for [story promo list](https://www.npmjs.com/package/@bbc/psammead-story-promo-list)

Reply:
I can see it was indeed used for `story promo list`. I wasn't on the discussion on the `alpha` releases just ignore the suggestions. Ideally it should be either `0.1.0-alpha` or `0.1.0-alpha.1`.